inc/query/query.php

```diff
@@ -160,111 +162,129 @@ public function handle_query_sibling( &$clauses, $id_column ) {
 	 * @param string $clauses   Query clauses.
 	 * @param array  $args   $WP_query args object.
 	 */
-	public function filter_query_clauses( &$clauses, $args ) {
-		global $wpdb;
-		if ( 1 >= count( $args ) ) {
-			return;
-		}
-		$relationships          = $args;
-		$relationships_operator = $clauses['relation'];
-		$join_on_clauses        = array();
-		$in_post_id             = array();
+	public function handle_multiple_relationship( &$clauses, $args ) {
+		$relationships = $args;
+		$objects       = array();
+		$object_ids    = array();
```

Comment:
$objects chứa các string là JOIN ON clause của term
$object_ids chứa các post_id là WHERE IN clause của post và user